src/mono/wasm/wasm.proj

```diff
@@ -53,6 +53,10 @@
   <UsingTask TaskName="CompileTimeZoneData"
     AssemblyFile="$(WasmBuildTasksAssemblyPath)"/>
   <Target Name="CompileTimeZones">
+    <PropertyGroup>
+      <FilterSystemTimeZones>false</FilterSystemTimeZones>
+    </PropertyGroup>
+
```

Comment:
Move this outside the target. Having this here means that it will always override the value set elsewhere.
Also, change this to: `<FilterSystemTimeZones Condition="'$(FilterSystemTimeZones)' == ''">false</FilterSystemTimeZones>`